genome_designer/main/xhr_handlers.py

```diff
@@ -645,6 +645,12 @@ def get_variant_list(request):
         variant_list = lookup_variant_result.result_list
         num_total_variants = lookup_variant_result.num_total_variants
 
+        # Truncate chromosme names and add a trailing '...' if longer
```

Comment:
This is one of the messiest/most complex pieces of code. I can help debug if it's unclear.